specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,268 @@
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/symssync/register": {
+      "put": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_Register",
+        "description": "Register files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SymsSync_Register.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "registerBody",
+            "required": true,
+            "description": "The register body",
+            "schema": {
+              "$ref": "#/definitions/registerObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/symssync/update": {
+      "put": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_Update",
+        "description": "Update files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SymsSync_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateBody",
+            "required": true,
+            "description": "The update body",
+            "schema": {
+              "$ref": "#/definitions/updateObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/symssync/delete": {
+      "put": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_Delete",
```

Comment:
>SymsSync_Delete [](http://example.com/codeflow?start=24&length=15)

The operationId will be used in SDK code generation.
Take "SymsSync_Delete" as example.
In C# SDK, we will have a class named "SymsSyncClient", inside the class, we will have a method named "Delete".

As I metioned before, we should avoid using abbreviation in SDK. So will it be better  to change operation id to "Metadata_Delete"?